### DISCUSSION TITLE
How is the NodeToolBar designed?

### DISCUSSION THREAD

**Discussion by lock4u** [bcakmakoglu/vue-flow#1722] [Q&A] [answered]
I do not understand why nodetoolbar need is-visible props!  How dose it works
and how to dynamic the nodetoolbar position

**Comment by bcakmakoglu** [comment, marked as answer]
> I do not understand why nodetoolbar need is-visible props!

What's the question here? It doesn't *need* it, it's there to allow for more flexibility and control when the toolbar is supposed to be visible.

> How does it work?

Look into the [source code](https://github.com/bcakmakoglu/vue-flow/tree/master/packages/node-toolbar) if you want to know about that.

>  and how to dynamic the nodetoolbar position

[Read the docs and look at the example for the toolbar.](https://vueflow.dev/examples/nodes/node-toolbar.html)
